docs/reference/ods-api-platform.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 2
+---
+
+# Ed-Fi ODS / API Platform
+
+The Ed-Fi Alliance publishes the [Ed-Fi Data
+Standard](./1-data-exchange/data-standard/), which models a broad spectrum of
+commonly exchanged and shared K-12 education data. The Ed-Fi ODS / API is a
+concrete implementation of a relational database and a companion API harmonized
+with the Ed-Fi Data Standard. The Ed-Fi ODS / API is tuned to the security and
+performance needs of K-12 organizations and the education technology vendors who
+serve them.
+
+The ODS implementation uses Microsoft SQL Server or PostgreSQL as a database
+platform. The API implementation is written in C#. Together, these form a data
+platform to host student-centric data. The ODS / API platform is vendor neutral,
+meaning that client applications may be written in any language.
+
+Please see the [Ed-Fi Technology Suite Supported
+Versions](./supported-versions.md) for an overview of which Data Standard
+version(s) are supported by a given ODS / API version.
+
+You can use the dropdown version control below to select which version's documentation
+you wish to view:
+
+import VersionDropDown from '@site/src/components/VersionDropDown';
+
+<VersionDropDown />
```

Comment:
I discovered that ODS/API was missing from the sidebar when looking at Admin API, MetaEd, etc. This page restores a page for ODS/API, and provides the version dropdown to help someone decide which version to visit. That way we don't need to hard code the versions here.

Reply:
Good catch! Not sure how that page / folder was removed. Even more strange is that /reference/ods-api is working still.

Not sure we have to address now, but I wonder how this will impact the folder structure when compiling docs for the next ODS/API release. Is the strategy to use `ods-api-platform.mdx` as the landing page and always link to ods/api versions from here?

I ask because there is some broken link behavior where selecting a version of the ODS/API from this page. The version number does not always get added to the links and results in a page not found.

Reply:
As soon as I commented I think I found the problem. Links ending with `readme.md` seem to have the versions inserted properly.

The `API Client Developers' Guide` clued me in as this is the only link on the page that works regardless of how you arrive on the page. Adding the `readme.md` resolved the others.

Eg. `./getting-started/` -> `./getting-started/readme.md`

I created a PR to merge into this one with the changes (Although I could have opted for making suggestions here)

https://github.com/Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io/pull/66

Reply:
> " Is the strategy to use ods-api-platform.mdx as the landing page and always link to ods/api versions from here?"

More or less, yes.